cmd/degitx/main.go

```diff
@@ -66,6 +66,10 @@ const pConfigUser = "${HOME}/.config/degitx/config.yml"
 
 const pConfigSys = "/etc/degitx/config.yml"
 
+/**
+ * @todo #74 Add channels for errors ans system signals and wait via select (back end)
```

Comment:
@stepanvalyavskiy I don't think it's a good idea to make multiple tasks for single one: all these tasks about channels and goroutins will depend on each other, so I'd create one single `@todo` for goroutins